addons/web_editor/static/src/js/editor/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -351,7 +351,11 @@ export class Powerbox {
                     this._context.filteredCommands = this._context.commands.filter(command => {
                         const commandText = (command.category + ' ' + command.name);
                         const commandDescription = command.description.replace(/\s/g, '');
-                        return commandText.match(fuzzyRegex) || commandDescription.match(exactRegex);
+                        const matched = commandText.match(fuzzyRegex) || commandDescription.match(exactRegex);
+                        if (matched) {
```

Comment:
```suggestion
                        if (commandText.match(fuzzyRegex) || commandDescription.match(exactRegex) || command.synonymKeywords?.some(keyword => exactRegex.test(keyword)) {
                              return true;
                        }
```

can be something like this.